publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dto/Publication.java

```diff
@@ -33,22 +38,26 @@
     public Publication(@JsonProperty("id") URI id,
                        @JsonProperty("institution_owner") URI institutionOwner,
                        @JsonProperty("doi") URI doi,
+                       @JsonProperty("doiRequest") DoiRequest doiRequest,
+                       @JsonProperty("modifiedDate") Instant modifiedDate,
```

Comment:
Do we have to worry about how objectMapper serialize these?

Reply:
Don't think so, the class has json properties defined.